src/bootstrap-public-path.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * Created by welchk on 7/20/16.
+ */
+__webpack_public_path__ = window.location.protocol + "//" + window.location.host + "/";
```

Comment:
Ah, I see that [__webpack_public_path__ is a magic global](https://github.com/webpack/webpack/issues/443).

I'd suggest renaming bootstrap-public-path to webpack-public-path and adding a comment inside that explains why its being set dynamically and includes a link to relevant issues and the docs that explain what  __webpack_public_path__ is.

Thanks so much for all your hard work on this Kyle! 💯